lib/vmpooler/api.rb

```diff
@@ -8,25 +8,24 @@ def initialize
       $config[:uptime] = Time.now
 
       # Set some defaults
-      $config[:redis] ||= Hash.new
+      $config[:redis] ||= {}
       $config[:redis]['server'] ||= 'localhost'
 
-      if ($config[:graphite]['server'])
+      if $config[:graphite]['server']
         $config[:graphite]['prefix'] ||= 'vmpooler'
       end
 
       # Connect to Redis
-      $redis = Redis.new(:host => $config[:redis]['server'])
+      $redis = Redis.new(host: $config[:redis]['server'])
```

Comment:
I guess?  I deployed the reformatted code to dev and everything seems to function identically AFAICT.